### ISSUE TITLE
Add filter student by class on home.

### ISSUE THREAD

**Issue by chrsep** [obserfy/obserfy#458]
If a school have a lot of students, it would be hard to parse through and find who you are looking for without a good filter.

#### TO-DO (each can be its own seperate PR)
- [x] Allow the schoolStore.GetStudents to accept classId on its parameter and filters students accordingly
- [x] Allow  `GET` `/schools/{schoolId}/students` to accepts class Id as query parameters
- [x] Update the useGetStudents hook on UI to accept classId
- [x] Add the class filter to UI.

#### UI wireframe
![excalidraw-2020314193352(6)](https://user-images.githubusercontent.com/8491552/86271945-021d9c80-bbf8-11ea-8d39-5046eb29afa6.png)

**Comment by zeirash** [comment]
this issue is for frontend or backend?

**Comment by chrsep** [comment]
both i guess, there are a few things to do. I'll update the description.
